### PR TITLE
(DOCSP-29111): Add info on how to use the Open API spec to create JSON config files

### PR DESCRIPTION
**Pull request by davidhou17**
Adds info to JSON page about using the Open API spec to create JSON configuration files.

[Jira](https://jira.mongodb.org/browse/DOCSP-29111)
[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-29111/reference/json/)
[Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=644c32b27efd2382b35f5c1f)

### CODE REVIEW

File: source/reference/json.txt [hunk 1]
```diff
@@ -1,8 +1,106 @@
...
+  or :ref:`upgrade a cluster <atlas-clusters-upgrade>`.
+- :ref:`Update a cloud backup schedule <atlas-backups-schedule-update>`.
+- :ref:`Create <atlas-clusters-search-indexes-create>` or 
+  :ref:`update an Atlas Search Index <atlas-clusters-search-indexes-update>`.
+
+Usage
+-----
+
+To learn which settings are accepted by the {+atlas-cli+}, view the request 
+body schema and request samples in the :ref:`Atlas Admin API Specification
```
Comment (sarahsimpers) [inline source/reference/json.txt:26]:
```suggestion
body schema and copy the request samples in the :ref:`Atlas Admin API Specification 
```

File: source/reference/json.txt [hunk 2]
```diff
@@ -1,8 +1,106 @@
...
+
+.. code-block:: sh
+   :caption: /example.json
+
+   {
+     "backupEnabled" : true,
+     "labels": [
+       {
+         "key": "foo",
+         "value": "bar"
```
Comment (sarahsimpers) [inline source/reference/json.txt:73]: Per the style guide/checked with Corry, in code examples we should use a placeholder here https://www.mongodb.com/docs/meta/style-guide/style/code-examples/
```suggestion
         "key": "<myKey>",
         "value": "<myValue>"
```

File: source/reference/json.txt [hunk 3]
```diff
@@ -1,8 +1,106 @@
...
+
+Usage
+-----
+
+To learn which settings are accepted by the {+atlas-cli+}, view the request 
+body schema and copy the request samples in the :ref:`Atlas Admin API Specification 
+<api-resources-spec>` for the endpoint that corresponds to the {+atlas-cli+} 
+command:
+
+.. list-table::
```
Comment (colm-quinn) [inline source/reference/json.txt:30]: [q/not blocking] This looks like a potential maintenance issue - if new commands are added with `--file` I could see it being easy to forget to update this page.

Should we aim for a more generic text here?

**Review by colm-quinn — APPROVED** [review]
LGTM, small question, not blocking. Thank you for making this page!

File: source/reference/json.txt (hunk 3 above)
Comment (davidhou17) [reply source/reference/json.txt:31]: Really great point - and thanks for bringing this up! I talked over Slack with @sarahsimpers and she's opening a ticket to think about ways to maintain this page (e.g an automated ticket generated if a file contains --file). For now, I've revised the text to be slightly more general/clarify that other commands might accept the `--file` option as well.

**Pull request merged**